packages/bootstrap/extra/dcos_internal_utils/cli.py

```diff
@@ -145,7 +145,9 @@ def get_roles():
 def main():
     opts = parse_args()
 
-    logging.basicConfig(format='[%(levelname)s] %(message)s', level='INFO')
+    # Display the pid in each log message to distinguish concurrent runs
+    log_format = '{}:[%(levelname)s] %(message)s'.format(os.getpid())
```

Comment:
Would it be useful to prefix pid with `pid` string for grepping purposes? It would be easier to grep logs for `pid:399` than just for number `399`.

Reply:
True, although grepping for `399:` will probably get it as well. (See my comment on keeping log length down).

Reply:
The `399:` has meaning for you right now, but has no meaning for customers and other developers that work with dcos. `bootstrap` script generates very little logs compared to other services so I'd prefer clarity over saving some bytes.

Zookeeper for example does:

```
Feb  3 03:43:20 ip-10-181-243-208 java: [myid:3] ...
``` 

For me it is much easier to understand what `myid` is if I compare this to nginx/lua logging output:

```
Feb  3 03:43:21 ip-10-181-243-208 adminrouter.sh: 2019/02/03 03:43:21 [info] 7469#0: *32461418 [lua] cache.lua:540: refresh_cache(): Executing cache refresh triggered by timer, context: ngx.timer
```

Where I'd have to lookup what `7469#0: *32461418` means.